app/controllers/comments_controller.rb

```diff
@@ -5,7 +5,11 @@ def create
 
     respond_to do |format|
       format.html do
-        redirect_to user_post_path(post.user.id, post.id) if comment.save
+        if comment.save
+          redirect_to user_post_path(post.user.id, post.id)
+        else
+          flash[:alert] = 'Error: comment is not published'
+        end
```

Comment:
- [ ] Kindly add flash message for successful comments

Reply:
Done✔